src/output.rs

```diff
@@ -81,8 +83,14 @@ pub fn emit(input: InputFile, template: &str, out_file: &mut dyn io::Write) -> a
 					newline(out, &indent)
 				},
 				Tag::CodeBlock(CodeBlockKind::Fenced(lang)) => {
-					let lang: &str = &lang;
-					let lang = match lang {
+					// Strip rustdoc code-block flags from the language
+					let mut lang = lang.to_string();
+					for flag in RUSTDOC_CODEBLOCK_FLAGS {
+						lang = lang.replace(flag, "");
+					}
+					lang = lang.replace(",", "");
```

Comment:
This code removes all `,` from the `lang` string, and the next match statement checks for `rust,` appearing in the same string. I assume the next statement can be stripped down to `if lang.is_empty() { lang = "rust".to_owned(); }`?

Reply:
Ah, good point.

Reply:
Pushed an update.